test/jasmine/tests/select_test.js

```diff
@@ -1299,10 +1299,77 @@ describe('@flaky Test select box and lasso per trace:', function() {
                 null, BOXEVENTS, 'violin select'
             );
         })
-        .catch(fail)
+        .catch(failTest)
         .then(done);
     });
 
+    ['ohlc', 'candlestick'].forEach(function(type) {
```

Comment:
So, apart from selection and the new legend items, can you list all the new features this PR adds to finance traces?

Reply:
.... mostly for changelog purposes :books: